app/views/state_file/questions/return_status/_rejected.html.erb

```diff
@@ -49,6 +49,6 @@
     <%= t('.contact_us') %>
   </section>
 
-  <% edit_return_path = params[:us_state] == 'az' ? az_questions_name_dob_path : ny_questions_name_dob_path %>
-  <%= link_to t('.edit_return'), edit_return_path, class: "button button--primary spacing-above-60" %>
+  <% edit_return_path = @error.correction_path || @error.default_correction_path %>
+  <%= link_to t('.edit_return'), "#{edit_return_path}?return_to_review=y", class: "button button--primary spacing-above-60" %>
```

Comment:
what does `return_to_review=y` do?

Reply:
is the `y` not significant because its just checking the presence of it at all?

Reply:
return_to_review is only checking presents - it means the prev / next step is data review rather than what is before / after the controller in the navigation